addons/web/static/src/core/notifications/notification.xml

```diff
@@ -2,14 +2,15 @@
 <templates xml:space="preserve">
 
     <t t-name="web.NotificationWowl" owl="1">
-        <div t-attf-class="o_notification {{className}}"
+        <div t-attf-class="o_notification toast show border-{{props.type}} overflow-hidden mb-2"
         role="alert" aria-live="assertive" aria-atomic="true">
-            <div class="o_notification_body">
-                <button type="button" class="close o_notification_close"
-                aria-label="Close" t-on-click="props.close">
-                    <span class="d-inline" aria-hidden="true">×</span>
+            <div t-attf-class="toast-header align-items-start justify-content-end pb-0 ">
+                <strong t-if="props.title" t-attf-class="o_notification_title text-{{props.type}} mr-auto" t-esc="props.title"/>
```

Comment:
Not sure I understand the advantage of adding a `justify-content-end` to position the close button and being able to conditionally render the `<strong t-if="props.title">` ?
Instead of just leaving the title in the DOM (even if empty) and the close will be position properly...